grpc/cancel.go

```diff
@@ -0,0 +1,20 @@
+package grpc
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// IsCanceled checks whether an error comes from an operation being canceled
+func IsCanceled(err error) bool {
+	if err == context.Canceled {
```

Comment:
Maybe use `errors.Is`?

Reply:
That would force consumers to upgrade to Go 1.13, I think.
Which is not a bad idea, but more work than I want to complete before merging this.
Do you want to file it as an issue/enhancement?

Reply:
That's a good consideration. I can file it as an enhancement